build.sbt

```diff
@@ -135,3 +144,30 @@ lazy val noPublishSettings =
     publish := (),
     publishLocal := ()
   )
+
+def runScriptedTask = Def.taskDyn {
+  val sbtBinVersion = (sbtBinaryVersion in pluginCrossBuild).value
+  val base = sbtTestDirectory.value
+  val testDirectoryFinder = base * AllPassFilter * AllPassFilter filter { _.isDirectory }
```

Comment:
Why does that find test directories?

Reply:
It works similar to the shell glob `.../sbt-test/*/*`